ts-sdk/whirlpool/src/increaseLiquidity.ts

```diff
@@ -422,6 +498,44 @@ export async function openFullRangePositionInstructions(
   );
 }
 
+/**
+ * Opens a new position in a concentrated liquidity pool within a specific price range.
+ * This function allows you to provide liquidity for the specified range of prices and adjust liquidity parameters accordingly.
```

Comment:
Might be nice to add as a note here that this function does not work for splash pools